addon/src/mocks/exposed-request-functions.js

```diff
@@ -404,10 +403,3 @@ export function mockDelete(...args) {
 
   return new MockDeleteRequest(modelName, { id, model });
 }
-
-/**
- Returns the Pretender instance used for the mocks.
- */
-export function getPretender() {
-  return RequestManager.getPretender();
-}
```

Comment:
too specific now that we allow other interceptors

get the interceptor you want via the request manager